packages/common/src/allo/backends/allo-v2.ts

```diff
@@ -198,10 +198,6 @@ export class AlloV2 implements Allo {
         60_000,
         publicClient
       );
-      await this.waitUntilIndexerSynced({
```

Comment:
Why though ? Is it going to be added on the cart page as a standalone call with wait and proceed logic ?

Reply:
it's just to stop people thinking that their vote hasn't gone through at checkout. i'd rather have people go to donation history page and not have the donation there since we have copy explaining that votes can could take a while to show up, user will be more assured that tx went through as opposed to failing at checkout.

Reply:
Do we wanna add a banner in the donation history pages stating, recent donations will appear shortly ? Else this just comes back as another bug report. Might as well get it done together. What do you think ?